mava/utils/environments/render_utils.py

```diff
@@ -22,322 +22,337 @@
 
 import numpy as np
 
-try:
+try:  # noqa
     os.environ["PYGAME_HIDE_SUPPORT_PROMPT"] = "hide"
     import pygame
     from pysc2.lib import colors, features, point, transform
     from pysc2.lib.renderer_human import _Surface
     from smac.env import StarCraft2Env
-except ModuleNotFoundError:
-    pass
 
+    class Renderer:
```

Comment:
I am not actually sure what these changes are. Maybe it is undoing one of your changes?

Reply:
O I think I reordered some of the logic because I wanted to make it not crash Mava if you do not use render and don't have the correct imports. But I am not sure if it is undoing some of your commits. Should I delete it completely for now?

Reply:
No worries.  I will merge and fix this in my PR.